app/uk/gov/hmrc/helptosavefrontend/models/HTSEvent.scala

```diff
@@ -76,13 +75,27 @@ case class ApplicationSubmittedEvent(auditSource: String, userInfo: NSIUserInfo)
   )
 }
 
-case class EligibilityCheckEvent(auditSource: String, nino: String, errorDescription: Option[String])(implicit hc: HeaderCarrier) extends HTSEvent {
+case class EligibilityResult(nino: NINO, errorDescription: Option[String])(implicit hc: HeaderCarrier) extends HTSEvent {
```

Comment:
in the case where the user is eligible we have a nice legible string we can put in the event - is this not something we want to put in? There are different reasons why the user is eligible

Reply:
fixed